extensions/php/languages/php/runnables.scm

```diff
@@ -0,0 +1,55 @@
+(
+    ; Class that follow the naming convention of PHPUnit test classes
+    ; and that doesn't have the abstract modifier
+    ; and have a method that follow the naming convention of PHPUnit test methods
+    ; and the method is public
+    (class_declaration
+        modifier: (_)? @modifier
+        (#not-eq? @modifier "abstract")
+        name: (_) @name
+        (#match? @name ".*Test$")
+        body: (declaration_list
+            (method_declaration
+                (visibility_modifier)? @visibility
+                (#eq? @visibility "public")
+                name: (_) @run
+                (#match? @run "^test.*")
+            )
+        )
+    )
+) @phpunit-test
+
+; Class that follow the naming convention of PHPUnit test classes
+; and that doesn't have the abstract modifier
+; and have a method that has the @test annotation
+; and the method is public
+(
+    (class_declaration
+        modifier: (_)? @modifier
+        (#not-eq? @modifier "abstract")
+        name: (_) @name
+        (#match? @name ".*Test$")
+        body: (declaration_list
+            ((comment) @comment
+                (#match? @comment ".*@test.*")
```

Comment:
Minor change: this should probably only match `@test` so that things like `@testdox`, `@testWith` or (eg) `@test-disabled` do not mark the annotated method as a test. (I checked and `@testdox` and `@testWith` do not imply that it's a test on their own; they must be paired with `@test` or `test_...`.)
 I suspect that something like this would take care of it for most use cases:
```suggestion
                (#match? @comment ".*@test\s.*")
```

Fun fact, I played with phpunit a little and found the following:
```php
/** @test */    ← works
/** @test*/     ← works
/**@test */     ← does not work
/** aaa@test */ ← works
/** @test       ← works (trimmed line/newline terminated)
 */
```
:shrug: